### DISCUSSION TITLE
Cannot reverify for student benefits

### DISCUSSION THREAD

**Discussion by rajeevkumar504** [community/community#52579] [GitHub Education]
Hello. I verified in  July 2022 that I am a student. My school was listed but didn't issue any student e-mail address, so I manually uploaded documents proving I am currently enrolled as a student.

I took a picture with my phone and uploaded it. I've tried 2 times now but I'm constantly getting rejected with the following:
<img width="315" alt="Screenshot 2023-04-12 at 8 46 41 AM" src="https://user-images.githubusercontent.com/97542658/231339567-56bc2631-c587-4f57-9438-1a50b29c9227.png">
They also wrote this:
"Try applying again after making more [contributions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/viewing-contributions-on-your-profile#what-counts-as-a-contribution) to the GitHub Classroom assignment(s) created by your teacher."
whereas I made many contributions in other repositories but not in the GitHub Classroom assignment(s) repository because I deleted that by mistake and didn't aware to complete the assignment.
If that is so important so please reassign me that assignment so that I can contribute and complete the assignment successfully and get verified my student benefits provided by GitHub because I will be a student till 2025.
Moderation Edit: I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Comment by rajeevkumar504** [comment]
Please give me a reply about it because my last Github pack has expired.
Please consider my case as soon as possible.

**Comment by samus-aran** [comment]
Hi @Rajeevjewar 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list. We often ask for specific types of evidences when we have evidence that your institution distributions them. I would suggest you reach out to your institution regarding to an academic email. 

> whereas I made many contributions in other repositories but not in the GitHub Classroom assignment(s) repository because I deleted that by mistake and didn't aware to complete the assignment.

That would be up to your teacher. We can not re-assign you to a repository.

**Reply by rajeevkumar504** [reply]
I uploaded the first preference evidence to approve my student developer pack but it still doesn't approve. I reached out to my institution mentor who says that there is no such email assigned to any student and they say you can upload your college ID card which is below.
![Idcard](https://user-images.githubusercontent.com/97542658/235344968-40fa2f3e-9ee3-4090-9d70-877abee5579e.jpg)
Please help me, I really need the tools to upskill and collaborate with me.
You can also guess if I was a student last year and I am doing engineering so I will be a student for the next three more years till 2025.
I had also uploaded my college ID card to prove that and the same ID card and they approve my student developer pack.

**Reply by rajeevkumar504** [reply]
Please reply to it.

**Comment by rajeevkumar504** [comment]
My github student pack doesn't approve till now.  What should I need to do?
Please tell me 🙏🏻.

**Comment by rajeevkumar504** [comment]
Please consider my case
Please Please..........
